README.md

```diff
@@ -78,17 +78,17 @@ A curated list of awesome Java frameworks, libraries and software.
 
 ## Ancients
 
-*In existence since the beginning of time and which will continue being used long after the hype has waned.*
+*In existence since the beginning of time, and which people will continue using long after the hype has waned.*
 
 * [Apache Ant](http://ant.apache.org/) - Build process management with XML.
-* [Apache Hadoop](http://hadoop.apache.org/) - Storage and large-scale processing of data-sets on clusters of commodity hardware.
+* [Apache Hadoop](http://hadoop.apache.org/) - Storage and large-scale processing of data sets on clusters of commodity hardware.
 * [Apache OpenNLP](https://opennlp.apache.org/) - Toolkit for common tasks like tokenization.
-* [Apache Velocity](http://velocity.apache.org/) - Templates for HTML pages, emails or source code generation in general.
+* [Apache Velocity](http://velocity.apache.org/) - A Java-based template engine for HTML pages, emails or source code generation.
```

Comment:
@LappleApple It's implied from it being in awesome-java that Velocity is Java-based, so would you kindly shorten it to "Template engine for...". :)